packages/clay-css/src/scss/atlas/variables/_navs.scss

```diff
@@ -4,6 +4,7 @@ $nav-font-size: 0.875rem !default;
 
 $nav-link-disabled-color: $gray-500 !default;
 
+$nav-link-line-height: 1.25rem !default; //20px
```

Comment:
Instead of adding a new line height variable we can place it directly in the menubar map.

Reply:
Sure! Thanks @pat270